app/src/main/java/ch/epfl/sdp/contamination/CachingDataSender.java

```diff
@@ -20,6 +20,8 @@
     String privateUserFolder = "privateUser/";
     String privateRecoveryCounter = "recoveryCounter";
 
+    int MAX_CACHE_ENTRY_AGE = InfectionAnalyst.UNINTENTIONAL_CONTAGION_TIME;

```

Comment:
isn't this useless?

Reply:
Nope, without it encapsulation would be violated. Even if right now it has the same value of UNINTENTIONAL_CONTAGION_TIME, the size of the cache is logically independent